src/config.rs

```diff
@@ -48,6 +48,11 @@ const DEFAULT_CLUSTER_ID: &str = "Kubernetes";
 
 const ISTIO_META_PREFIX: &str = "ISTIO_META_";
 
+/// Fetch the XDS/CA root cert file path based on below constants
+const XDS_ROOT_CERT_PROVIDER_ENV: &str = "XDS_CERT_PROVIDER";
```

Comment:
I would name it XDS_ROOT_CA and CA_ROOT_CA to match the istio-agent

Reply:
sure, it makes sense to match the isito-agent